### ISSUE TITLE
`LDY #Pad_Input` is better thought of as `LDY #PLAYER_TOPWALKSPEED`

### ISSUE THREAD

**Issue by velipso** [captainsouthbird/smb3#4]
https://github.com/captainsouthbird/smb3/blob/c1f3d94f66e5c5a7c70b2a2aae4a0e13a429fed7/PRG/prg008.asm#L2347

This code confused me until I realized that `Pad_Input` is `$18`, which means the code should be `LDY #PLAYER_TOPWALKSPEED` (imo).

**Comment by captainsouthbird** [comment]
Yup, that was an old leftover error from earlier times when I was just mass replacing labels, fixed it, sorry I wasn't paying attention sooner
